go-controller/cmd/ovn-kube-util/app/readiness-probe.go

```diff
@@ -25,7 +30,7 @@ var callbacks = map[string]readinessFunc{
 	"ovnsb-db-raft":  ovnSBDBRaftReadiness,
 }
 
-func ovnControllerReadiness(target string) error {
+func ovnControllerReadiness(nbClient libovsdbclient.Client, sbClient libovsdbclient.Client, target string) error {
```

Comment:
In some of these function we don't use the passed clients, but because of how this was designed It's easier for now to just pass all the clients to each readiness function